packages/s3-cache/src/index.ts

```diff
@@ -19,6 +20,11 @@ export class S3Cache implements CacheStrategy {
     this.bucketName = bucketName
   }
 
+  buildTagKeys(tags?: string | string[]) {
+    if (!tags || (Array.isArray(tags) && !tags.length)) return ''
```

Comment:
```suggestion
    if (!tags?.length)) return ''
```
it should be enough for both cases with array and string

Reply:
yes, thank you 
I have fixed it